RHDEVDOCS-2815 Authenticating pipelines using git secret (#37610)
openshift/openshift-docs: modules/op-using-secrets-as-a-nonroot-user.adoc

// This module is included in the following assembly:
//
// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc

[id="op-using-secrets-as-a-nonroot-user_{context}"]
= Using secrets as a non-root user

> You may need to use secrets as a non-root user in certain scenarios, such as:

Reviewer: ```suggestion
You might need to use secrets as a non-root user in various scenarios, such as:
```
